src/core/openthread-core-default-config.h

```diff
@@ -1984,4 +1984,14 @@
 #endif // OPENTHREAD_CONFIG_RADIO_2P4GHZ_OQPSK_SUPPORT
 #endif // OPENTHREAD_CONFIG_DEFAULT_CHANNEL
 
+/**
+ * @def OPENTHREAD_CONFIG_ENABLE_VERIFY_ACK_FP_FLAG
+ *
+ * Define as 1 to enable verify ACK_FP flag.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_ENABLE_VERIFY_ACK_FP_FLAG
+#define OPENTHREAD_CONFIG_ENABLE_VERIFY_ACK_FP_FLAG 0
```

Comment:
How about a radio capability instead of this?
 
If radio does not provide capability the OT code should set the `mAckWithFramePending` to some default (probably assume `true`).

Reply:
Talked offline about this. Let's go for the feature.